schemas/BaseElements.xsd

```diff
@@ -571,9 +571,9 @@
 												<xs:element minOccurs="0" name="RoofColor"
 												type="WallAndRoofColor"/>
 												<xs:element minOccurs="0" name="SolarAbsorptance"
-													type="SolarAbsorptance"/>
+												type="SolarAbsorptance"/>
```

Comment:
I've seen these whitespace changes in a couple PRs now. It might be some auto formatting that OxygenXML does. Hopefully this doesn't cause conflicts.

Reply:
Yes. Probably best to apply the Oxygen formatting to master and then merge it into all the PRs to make the whitespace diffs go away.